.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           ./gomplate -f .github/runner/github-runner-deployment.yaml.tmpl -o runner_deployment.yaml
 
       # Setup gcloud CLI
-      - uses: GoogleCloudPlatform/github-actions@0.1.2
+      - uses: GoogleCloudPlatform/github-actions@e23988b2af9696c66e87d1efbc688d3a80c3be14
         with:
```

Comment:
do we want to set the version here as well or no?

Reply:
No, the latest tagged version is tooo old that's why we use git sha :(

Reply:
I meant a `version` key under `with:` 😄

Reply:
Aaaa, I see 😄  a good point!